### ISSUE TITLE
Mozilla Deepspeech  epochs

### ISSUE THREAD

**Issue by YasineNifa** [mozilla/DeepSpeech#2116]
Hello !!

I want to train the Mozilla Deepspeech 's model with epochs = 100 
But when I execute the following command :
`python -u DeepSpeech.py  --epochs 100 ` 
the execution stops at the epoch number 20 and gives me as output the model 
May I have change something in the Deepspeech.py script ?

**Comment by YasineNifa** [comment]
Do I have to comment this part of code :
`# Early stopping
                    if FLAGS.early_stop and len(dev_losses) >= FLAGS.es_steps:
                        mean_loss = np.mean(dev_losses[-FLAGS.es_steps:-1])
                        std_loss = np.std(dev_losses[-FLAGS.es_steps:-1])
                        dev_losses = dev_losses[-FLAGS.es_steps:]
                        log_debug('Checking for early stopping (last %d steps) validation loss: '
                                  '%f, with standard deviation: %f and mean: %f' %
                                  (FLAGS.es_steps, dev_losses[-1], std_loss, mean_loss))
                        if dev_losses[-1] > np.max(dev_losses[:-1]) or \
                           (abs(dev_losses[-1] - mean_loss) < FLAGS.es_mean_th and std_loss < FLAGS.es_std_th):
                            log_info('Early stop triggered as (for last %d steps) validation loss:'
                                     ' %f with standard deviation: %f and mean: %f' %
                                     (FLAGS.es_steps, dev_losses[-1], std_loss, mean_loss))
                            break`

**Comment by lissyx** [comment]
@YasineNifa You should just pass `--noearly_stop`
